src/gui_tool/entry.py

```diff
@@ -45,8 +49,14 @@ def split_file(file_loc, mdi:MetaDataItem):
         try:
             context = GUIContext(file_loc)
             gui = SPGUIManager(context)
-            gui.start()
-            return mdi
+            rs = gui.start()
+            ret = []
+            for start, end in rs:
+                m = mdi.clone()
+                m.start_i = start
+                m.end_i = end
```

Comment:
Could you set the "is_split_url" for each metadata item if there is more than one (start, end) in rs and can the first element in ret be mdi and the rest clones, otherwise we will lose track of that exact metadata item and it will permanently be stuck in progress.